R/UK.R

```diff
@@ -459,22 +481,25 @@ UK <- R6::R6Class("UK", # rename to country name
         )
     },
 
-    #' @description Get lookup table for UK authorities
+    #' @description Download lookup table for UK authorities
     #' @importFrom vroom vroom col_character
-    #' @importFrom dplyr select distinct filter bind_rows arrange
-    #' @importFrom tidyr drop_na
-    #' @importFrom tibble tibble
-    get_authority_lookup_table = function() {
-      authority_data <- vroom(
+    download_authority_data = function() {
+      self$authority_data <- vroom(
```

Comment:
does this change? Should be just put this in the package as a dataset?

Reply:
If it never changes that would probably make more sense

Reply:
I don't really know but my guess is it changes fairly slowly....

Reply:
open to suggestions either way.

Reply:
I will leave this for now and get we can impliment a change when the new region codes frame work is in place. #200